app/assets/javascripts/component_guide/accessibility-test.js

```diff
@@ -132,15 +132,53 @@
         var activeElParent = _findParent(activeEl, '[data-module="test-a11y"]')
         var wrapper = activeElParent.querySelector(resultContainerSelector)
 
-        if (wrapper) {
-          var resultHTML = '<h3>(' + result.id + ') ' + result.summary + ' <a href="' + result.url + '">(see guidance)</a></h3>' +
-          '<p>' + selectorObj.reasons.join('<br />') + '</p>' +
-          '<p>Element can be found using the selector:<br /><span class="selector">' +
-          selectorObj.selector +
-          '</span></p>'
-
-          wrapper.insertAdjacentHTML('beforeend', resultHTML)
+        if (!wrapper) {
+          return
         }
+
+        // Section to announce the overall problem.
+        var headerNodeLink = document.createElement('a')
```

Comment:
Might be better to use `<template>` rather than constructing the DOM like this here...

Reply:
Indeed, but does the job for now. Thanks for doing this work!